packages/components/src/components/install-guide/ProjectPickerRow.vue

```diff
@@ -95,23 +95,20 @@
           </ol>
         </template>
         <template v-if="isJS">
-          <h3 class="install-heading">Run the AppMap Installer</h3>
-          To make AppMaps of your JavaScript project, you need to install the
-          <code class="inline">appmap-agent-js</code>
-          package from NPM and configure your project to use it. We provide an open source
-          installer, or you can install manually. Advantages of using the installer include:
-          <ol>
-            <li>Verifies that your Node.js version is supported by AppMap.</li>
-            <li>
-              Verify that your Express, Jest and Mocha versions (if present) are supported by
-              AppMap.
-            </li>
-            <li>
-              Detects and supports <i>package-lock.json</i> (npm) and <i>yarn.lock</i> (yarn).
-            </li>
-            <li>Creates the configuration file <i>appmap.yml</i>.</li>
-            <li>Has built-in support if you encounter any problems.</li>
-          </ol>
+          <h3 class="install-heading">Use the <i>appmap-node</i> package</h3>
+          <p>
+            Using <i><a href="https://github.com/getappmap/appmap-node">appmap-node</a></i> is the
+            recommended way to record Node.js applications. It's suitable for most Node.js projects
+            and includes support for Express, Nest.js, Next.js, Next.js, Jest, Mocha and Vitest. You
+            can proceed directly to the next step; <i>appmap-node</i> does not require explicit
+            installation.
```

Comment:
This reminds me, we need to update the `appmap install` command to stop installing `appmap-agent-js`. 

https://github.com/getappmap/appmap-js/issues/1566